Dockerfile

```diff
@@ -1,192 +1,170 @@
+COPY --from=node-builder ${APP_HOME} ${APP_HOME}
 
-# Run Django's collectstatic to collect assets from the frontend build.
-#
-# Our Django settings file requires a SECRET_KEY, but we don't want to
-# bake our key into the Docker image. We need to provide one in order to
-# be able to run collectstatic, even though the key value isn't actually
-# used in any way during staticfiles collection. We provide a random
-# secret key here for this step only.
-RUN SECRET_KEY=only-for-collectstatic cfgov/manage.py collectstatic
-
-#######################################################################
-# Build mod_wsgi against target Python version
-FROM base as cfgov-mod-wsgi
-WORKDIR /tmp
-RUN apk add --no-cache --virtual .build-deps apache2-dev gcc make musl-dev
-RUN wget https://github.com/GrahamDumpleton/mod_wsgi/archive/refs/tags/4.9.0.tar.gz -O mod_wsgi.tar.gz
-RUN echo -n "0a6f380af854b85a3151e54a3c33b520c4a6e21a99bcad7ae5ddfbfe31a74b50  mod_wsgi.tar.gz" | sha256sum -c
-RUN tar xvf mod_wsgi.tar.gz
-RUN cd mod_wsgi* && ./configure && make install
-RUN ls /usr/lib/apache2/mod_wsgi.so  # Ensure it compiled and is where expected
-RUN apk del .build-deps
-RUN rm -Rf /tmp/mod_wsgi*
+# Run our initial data entrypoint script
+ENTRYPOINT ["./docker-entrypoint.sh"]
 
-#######################################################################
-# Production-like Apache-based image
-FROM base as cfgov-prod
+# Give our user ownership over the app directory
+RUN chown -R ${USERNAME}:${USERNAME} ${APP_HOME}
 
-ENV HTTPD_ROOT /etc/apache2
+# Run the application with the user we created
+USER $USERNAME
 
-# Apache HTTPD settings
-ENV APACHE_SERVER_ROOT ${APP_HOME}/cfgov/apache
-ENV APACHE_PROCESS_COUNT 4
-ENV ACCESS_LOG /dev/stdout
-ENV ERROR_LOG /dev/stderr
-ENV STATIC_PATH ${APP_HOME}/cfgov/static/
-ENV LIMIT_REQUEST_BODY 0
+# Run Django's runserver
+CMD python ./cfgov/manage.py runserver 0.0.0.0:8000
 
-# mod_wsgi settings
-ENV CFGOV_PATH ${APP_HOME}
-ENV CFGOV_CURRENT ${APP_HOME}
-ENV PYTHONPATH ${APP_HOME}/cfgov
+#######################################################################
+# Production runs with Django via Gunicorn with cfgov.settings.production
+FROM python AS prod
 
 # Django Settings
 ENV DJANGO_SETTINGS_MODULE cfgov.settings.production
+ENV STATIC_PATH ${APP_HOME}/cfgov/static/
 ENV DJANGO_STATIC_ROOT ${STATIC_PATH}
 ENV ALLOWED_HOSTS '["*"]'
 
-# Install Apache server and curl (container healthcheck),
-# and converts all Docker Secrets into environment variables.
-RUN apk add --no-cache apache2 curl && \
-    echo '[ -d /var/run/secrets ] && for s in $(find /var/run/secrets -type f -name "*" -maxdepth 1) ; do export $s=$(cat $s); done && cd -' > /etc/profile.d/secrets_env.sh
-
-# Link mime.types for RHEL Compatability in apache config.
-# TODO: Remove this link once RHEL is replaced
-RUN ln -s /etc/apache2/mime.types /etc/mime.types
+# Copy the application code over
+COPY cfgov ./cfgov/
+COPY static.in ./static.in/
```

Comment:
Is `static.in` needed anywhere any longer? We used to use it for bringing in font files when they were kept in a private repo, but we no longer do this. Just noting it here but that's something we could clean up in a subsequent PR.

Reply:
I'm not certain, but [it is configured](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/cfgov/settings/base.py#L298-L301) and [contains our font files](https://github.com/cfpb/consumerfinance.gov/tree/main/static.in), so I think we still need to include it here with changes there.